ochk/sdk/client.go

```diff
@@ -129,8 +130,9 @@ func NewClient(ctx context.Context, host string, tenant string, username string,
 			service:    authClient.Routers,
 		},
 		VirtualMachines: VirtualMachinesProxy{
-			httpClient: httpClient,
-			service:    authClient.VirtualMachines,
+			httpClient:    httpClient,
+			service:       authClient.VirtualMachines,
+			legacyService: authClient.VirtualMachinesnsx,
```

Comment:
to legacy serwis będzie niebawem do wywalenia? Można dać komentarz //DEPRECATED, wtedy przekreśla użycie